.github/workflows/ci.yml

```diff
@@ -24,10 +24,11 @@ jobs:
           cache: "yarn"
       - name: 'Check or update Yarn cache (fix w/ "yarn install")'
         env:
+          YARN_ENABLE_HARDENED_MODE: true # enable hardened mode as CI may run on untrusted repos
```

Comment:
Might want to disable it in the other jobs that also run an install to avoid slowing down the CI.

Reply:
Thanks. I disable it in the `.yarnrc.yml`. I expect the env config here should overwrite the `.yarnrc.yml`, which will then overwrite yarn's CI hardened mode switch, right?

Reply:
Yes, that's correct.